CMakeLists.txt

```diff
@@ -1,3 +1,8 @@
+cmake_minimum_required(VERSION 3.12)
+
+project(arbalest)
```

Comment:
no, it's RT3 (see main CMakeLists.txt)

Reply:
We did compile differently.  I see now why you said it's RT3..  I made arbalest's build work independently, not as a subdir in the rt-cubed repo.  I would think that's far more preferable to having to hand-edit the rt-cubed/src/CMakeLists.txt file... no?